From ae50905502d2f9226736620c16e0182b7ff7528d Mon Sep 17 00:00:00 2001
From: Richard Feliciano <rfeliciano@BeaconEmbedded.com>
Date: Mon, 4 Nov 2024 12:17:34 -0600
Subject: [PATCH] Modified to build wifi for mlan utils

Added the nxp-wlan-sdk but but needed to resolve an undefined
flag

Signed-off-by: Richard Feliciano <rfeliciano@BeaconEmbedded.com>

diff --git a/mlinux/moal_sta_cfg80211.c b/mlinux/moal_sta_cfg80211.c
index f13f641..5d0a29b 100644
--- a/mlinux/moal_sta_cfg80211.c
+++ b/mlinux/moal_sta_cfg80211.c
@@ -10134,7 +10134,7 @@ mlan_status woal_register_cfg80211(moal_private *priv)
 		PRINTM(MIOCTL, "Follow countryIE provided by AP.\n");
 	}
 #endif
-#if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 19, 0)
+#if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 19, 0) && (CFG80211_VERSION_CODE < KERNEL_VERSION(6, 1, 1))
 	/*REGULATORY_IGNORE_STALE_KICKOFF: the regulatory core will _not_ make
 	 * sure all interfaces on this wiphy reside on allowed channels. If this
 	 * flag is not set, upon a regdomain change, the interfaces are given a
-- 
2.34.1

